src/libstd/prelude/v1.rs

```diff
@@ -123,3 +123,6 @@ pub use crate::string::{String, ToString};
 #[stable(feature = "rust1", since = "1.0.0")]
 #[doc(no_inline)]
 pub use crate::vec::Vec;
+#[unstable(feature = "hashmap_prelude", issue = "0")]
+#[doc(no_inline)]
+pub use crate::collections::HashMap;
```

Comment:
There's a comment further up saying this is equivalent to `liballoc/prelude.rs`. First that file became a directory, but also the comment is not correct any more with this change.